setup.py

```diff
@@ -22,6 +22,13 @@
     ]
 
 
+extras_require = {
+    'tests': tests_require,
+    'deletepermission': [
+        'collective.deletepermission',
```

Comment:
i would, for the sake of consistentcy, also assign the `deletepermission` extra to a variable